library/Freckle/App/Prelude.hs

```diff
@@ -135,7 +139,7 @@ import Data.Either (partitionEithers)
 import Data.Foldable as Foldable hiding (foldl1, foldr1)
 import Data.Maybe
   (catMaybes, fromMaybe, isJust, isNothing, listToMaybe, mapMaybe, maybeToList)
-import qualified Data.Text as T
+import Data.Text.Encoding (decodeUtf8, encodeUtf8)
```

Comment:
I might've mentioned it in the discussion, but for the record:

`RIO` and any others that follow the "no unsafe functions by default" philosophy don't export this, since it's technically unsafe / lossy. They doe export `decodeWith` and `lenientDecode` so you can make it easily-but-explicitly yourself.

I'm conflicted on proposing we do that -- One of the very first thing I add to my `${App}.Prelude` in an `RIO` project is define a `decodeUtf8`, but figured I'd raise it.

Separate form that, what about _also_ adding `decodeUtf8With` and the various `.Errors` options? The `Wait` module in this project itself is one example that would benefit.